### PR TITLE
enable cd for appdome-build-2secure plugin

### PR DESCRIPTION
**Pull request by idanhauser**
- [X ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ X] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

I have followed this guide https://www.jenkins.io/doc/developer/publishing/releasing-cd/

### CODE REVIEW

File: .github/workflows/cd.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
+
+name: cd
+on:
+  workflow_dispatch:
+#  check_run:
+#    types:
+#      - completed
```
Comment (NotMyFault) [inline .github/workflows/cd.yaml]:
```suggestion
  check_run:
    types:
      - completed
```
This is needed for CD.

**Review by NotMyFault — CHANGES_REQUESTED** [review]
You're still lacking changes to the pom.xml.

**Comment by idanhauser** [comment]
> You're still lacking changes to the pom.xml.


@NotMyFault, thanks for your help. I want to set the versioning of my plugin similar to that of the Github plugin (https://github.com/jenkinsci/github-plugin). Therefore, I have been referring to their pom.xml file to adopt their versioning approach. Is there a way to ensure that my versioning style matches theirs?

**Comment by NotMyFault** [comment]
>  I want to set the versioning of my plugin similar to that of the Github plugin (https://github.com/jenkinsci/github-plugin)

The GH plugin does not cut released via CD. CD aims to cut releases automatically, depending on the merge of PRs with interesting labels.
If you want to follow a versioning scheme like the GH plugin, CD likely not what you want. Instead, refer to the [maven release plugin](https://www.jenkins.io/doc/developer/publishing/releasing-manually/).

**Comment by idanhauser** [comment]
> > I want to set the versioning of my plugin similar to that of the Github plugin (https://github.com/jenkinsci/github-plugin)
> 
> The GH plugin does not cut released via CD. CD aims to cut releases automatically, depending on the merge of PRs with interesting labels. If you want to follow a versioning scheme like the GH plugin, CD likely not what you want. Instead, refer to the [maven release plugin](https://www.jenkins.io/doc/developer/publishing/releasing-manually/).

Thanks, @NotMyFault! If I choose to proceed with that course of action, I'll close the PR afterwards. Just to confirm, if I've added the encrypted password to the ~/.m2/settings.xml file like the guide says, then all I need to do to release the plugin is run the following command: "mvn release:prepare release:perform," and the plugin will be released, right?

**Comment by NotMyFault** [comment]
Yes, that's all you need. If in doubt, feel free to visit the docs or ask on Gitter/matrix how other maintainers release their plugins

**Comment by idanhauser** [comment]
Thank you !
For now, I will go with the manual release process. Therefore, I will close both this pull request and the other one found at https://github.com/jenkins-infra/repository-permissions-updater/pull/3183.

**Pull request closed without merging**